notebooks/kbmod_analysis_demo.ipynb

```diff
@@ -38,92 +38,252 @@
     "4) Once you have the results you can use the steps in this notebook to create plots with postage stamps and lightcurves side-by-side for a final look by eye.\n",
     "  You will first have to update some variables to match the arguments you chose for example_search.py in step 3.\n",
     "  * Change results_dir to match --results_dir option.\n",
-    "  * Change results_suffix to match --results_suffix option."
+    "  * Change results_suffix to match --results_suffix option.\n",
+    "  \n",
+    "Note that fake result data (corresponding to the `data/demo` images) is included in `data/fake_results` with the suffix DEMO."
```

Comment:
This is a really good point. This PR is meant to be an initial step to refactoring all of the file code out of CreateStamps. I would also like to move the neural network filtering out of create stamps, so we can move create stamps out of analysis. Do you think it is worth doing that in this PR or following up with later PRs? I would like to keep the scope of this PR reasonably manageable, but happy to change it if it would be cleaner.

Reply:
I moved `create_stamps.py` out of analysis. It was not using any additional dependencies (just matplotlib which is already imported).

I want to refactor and rewrite a lot of `create_stamps.py` to make it more readable, but I will leave that for a future change.

Reply:
I think it's appropriate to split this in different PRs. I think the crucial feature to consider here, except for ensuring the code is correct of course, is protecting the imports for packages that don't exist so that the installation and import is reasonably snappy, because (my impression is) we would be installing+running kbmod as part of a batch processing procedure and would only need additional packages on the main compute resource where we also do analysis of results. 

A note for later would also be to replace the [Pandas DataFrame](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html) with [AstroPy Table](https://docs.astropy.org/en/stable/table/index.html) class. The two should be nearly identical in functionality, see [differences](https://docs.astropy.org/en/stable/table/pandas.html). I mention this here because Pandas is a big package and I think it's only really used in analysis to more easily deal (compared to numpy's structured or recarrays) with columnar data I think.